src/components/Navigation/Navigation.tsx

```diff
@@ -34,7 +33,7 @@ export default class Navigation extends React.Component<Props, never> {
     }
   }
 
-  getChildContext() {
+  get getContext() {
```

Comment:
`getContext` makes me think this is a function, and `get getContext` feels odd. Should this just be `get context` then accessed with this.context?

Should this be private?

Reply:
`this.context` feels a bit too similar (the same as) the old context API, though I agree that the double `get` is a bit semantically weird